src/vstest.console/Resources/Resources.resx

```diff
@@ -277,7 +277,7 @@
     <value>'{0}' not found.</value>
   </data>
   <data name="Framework35NotSupported" xml:space="preserve">
-    <value>Framework35 is not supported. For projects targeting .Net Framework 3.5, please use Framework40 to run tests in CLR 4.0 "compatibility mode".</value>
+    <value>Framework35 is not supported. For projects targeting .Net Framework 3.5, tests are running in "compatibility mode".</value>
```

Comment:
Framework35 is not supported. For projects targeting .Net Framework 3.5, test will run in CLR 4.0 "compatibility mode".